build-tests-subspace/rush-sdk-test/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rush-sdk-test",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A minimal example project that imports APIs from @rushstack/rush-sdk",
+  "license": "MIT",
+  "scripts": {
+    "build": "heft build --clean",
+    "start": "node lib/start.js",
+    "_phase:build": "heft run --only build -- --clean"
+  },
+  "dependencies": {
+    "@rushstack/rush-sdk": "workspace:*",
+    "colors": "~1.2.1"
```

Comment:
I recently removed `colors` from the rest of the repo. Can you use `Colorize` from `@rushstack/terminal` instead?